pkg/proxy/iptables/proxier.go

```diff
@@ -404,6 +405,8 @@ var iptablesCleanupOnlyChains = []iptablesJumpChain{
 	{utiliptables.TableFilter, kubeServicesChain, utiliptables.ChainInput, "kubernetes service portals", []string{"-m", "conntrack", "--ctstate", "NEW"}},
 }
 
+var iptablesCleanupPeriod = time.Minute
```

Comment:
what if we only do the cleanup during the `syncPeriod` ?

```
iptables:
  masqueradeAll: false
  masqueradeBit: 14
  minSyncPeriod: 1s
  syncPeriod: 30s
```

this way it can be parameterizable

Reply:
makes sense